specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-preview.1/TextAnalytics.json

```diff
@@ -568,13 +581,17 @@
             "neutral",
             "negative",
             "mixed"
-          ]
+          ],
+          "x-ms-enum": {
+            "name": "DocumentSentimentValue",
+            "modelAsString": false
+          }
         },
         "statistics": {
           "$ref": "#/definitions/DocumentStatistics"
         },
         "documentScores": {
-          "description": "Document level sentiment confidence scores between 0 and 1 for each sentiment class."
```

Comment:
You can leave a description here and below still, otherwise the interface method won't have a comment in the reference docs.

Reply:
@xirzec Sure. Thanks for the suggestion. Updated. 
Could you approve this PR if everything looks good? Thanks!